src/Sarif.UnitTests/ConvertToSchemaUriTests.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
Just a couple of formatting issues:

* All our source files need a copyright/license comment at the top. Copy one from an existing file and paste it in here.
* We do sort the `using` directives into alphabetical order, but we place the `System.*` usings first. (I don't know why ;-)) In VS, you can just right-click anywhere in the using directives, and then select menu item Organize usings, Sort usings.
* Remove the extra blank line before the first test method.